Sources/Channel.swift

```diff
@@ -27,18 +27,18 @@ import Swift
 public class Channel {
     
     /// The topic of the Channel. e.g. "rooms:friends"
-    let topic: String
+    public let topic: String
     
     /// The params sent when joining the channel
     var params: Payload
 
     /// The Socket that the channel belongs to
-    weak var socket: Socket?
+    public weak var socket: Socket?
 
     
     
     /// Current state of the Channel
-    var state: ChannelState
+    private(set) public var state: ChannelState
```

Comment:
This breaks some tests. Also, you can access the state of the channel already by calling `isClosed`, `isJoined`, etc. See [more here](https://github.com/davidstump/SwiftPhoenixClient/blob/796d7fa7a08b9fd781bc712f2df423d4b2c73dfb/Sources/Channel.swift#L357). So I don't think this property needs to be exposed

Reply:
OK I can live without this